File: MarsRoverProb/src/com/marsroverprob/commands/MoveCommand.java
package com.marsroverprob.commands;

import com.marsroverprob.helper.RoverHelper;
import com.marsroverprob.model.Rover;

public class MoveCommand implements RoverCommand {
  public void execute(Rover rover, String maxPos) {
    RoverHelper.moveAhead(rover,maxPos);
  }
}

File: MarsRoverProblem_C#/Source/marsroverprob/commands/MoveCommand.cs
using System;

using marsroverprob.commands.intf;
using marsroverprob.model;
using marsroverprob.helper;

namespace marsroverprob.commands.movecommand
{
    class MoveCommand : IRoverCommand {
        public void execute(Rover rover, String maxPos) {
            RoverHelper.moveAhead(rover,maxPos);
        }
    }
}